examples/bri-3/src/bri/workgroup/workgroups/models/workgroup.ts

```diff
@@ -1,93 +1,86 @@
 import { Workstep } from '../../worksteps/models/workstep';
-import { IWorkgroup } from './workgroup.interface';
 import { Workflow } from '../../workflows/models/workflow';
 import { BpiSubject } from '../../../identity/bpiSubjects/models/bpiSubject';
-import { Security } from '../../../policy/models/security';
-import { Privacy } from '../../../policy/models/privacy';
 
-export class Workgroup implements IWorkgroup {
-  private id: string; // TODO: Add uuid after #491
-  private name: string;
-  private administrator: BpiSubject[];
-  private securityPolicy: Security[]; //TODO Implement securityPolicy #485
-  private privacyPolicy: Privacy[]; //TODO Implement privacyPolicy #485
-  private participants: BpiSubject[];
-  private worksteps: Workstep[];
-  private workflows: Workflow[];
+export class Workgroup {
+  private _id: string; // TODO: Add uuid after #491
```

Comment:
I think the agreement is that we ditch private and _ in var names in order to make sure mapping from\to prisma models work? @Manik-Jain

Reply:
Yes

Reply:
private, _, and getters removed